pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -25,10 +25,11 @@ var (
 )
 
 type fileReader struct {
-	Cfg              *DashboardsAsConfig
-	Path             string
-	log              log.Logger
-	dashboardService dashboards.DashboardProvisioningService
+	Cfg                          *DashboardsAsConfig
+	Path                         string
+	log                          log.Logger
+	dashboardProvisioningService dashboards.DashboardProvisioningService
+	fileFilterFunc               func(fileInfo os.FileInfo) bool
```

Comment:
Since this func is only used for one test I think it would be better to just do file operations on disk during tests.
It's quite common in Go since compiling/tests is so fast.

Reply:
I realised I do not need the function after all, was able to setup the test without it.